apps/arweave/src/ar_http_iface_client.erl

```diff
@@ -370,7 +370,7 @@ get_sync_record(Peer) ->
 		peer => Peer,
 		method => get,
 		path => "/data_sync_record",
-		timeout => 180 * 1000,
+		timeout => 30 * 1000,
```

Comment:
Since we are working with timeouts here, maybe we can move it into configuration as well?.. Such things might be tuned by technically advanced miners to improve performance.

Reply:
I created an [issue](https://github.com/ArweaveTeam/arweave-dev/issues/845), I suggest we take care of it separately